shub/utils.py

```diff
@@ -39,6 +39,24 @@
 STDOUT_ENCODING = sys.stdout.encoding or FALLBACK_ENCODING
 LAST_N_LOGS = 30
 
+_SETUP_PY_TEMPLATE = """\
```

Comment:
Moved this change into https://github.com/scrapinghub/shub/pull/250, it's more consistent to change it there and rebase this PR on top of ot